packages/messages-react/src/common/MessageIcon.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { messageTypes } from "./types.d";
+
+interface Props {
+    messageType: messageTypes;
+}
+
+export function MessageIcon({ messageType }: Props) {
+    switch (messageType) {
+        case "error":
+            return (
+                <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Nå _må_ vi få på plass en pakke for dette.

Reply:
Designerne holder på med en gjennomgang av ikonene som brukes, så tenker vi lager en pakke når de har fått oversikt og evt. justert så ting henger mer sammen.